release.sh

```diff
@@ -104,7 +104,7 @@ git checkout --detach
 
 # Build generated files and add them to the repository (for bower)
 git clean -fdx build dist
-make setup dist
+npm run dist
```

Comment:
Should this command run when `--dry-run`?

Reply:
I think so.  It won't do anything other than echo the command to the shell which is what we want in the dry run.

Reply:
@kevinbarabash Do you mean it should actually build the KaTeX or not?

Reply:
Sorry, the real command should not run, it should jus echo what the command to the console.